### DISCUSSION TITLE
thumbtack icon

### DISCUSSION THREAD

**Discussion by kcantwel** [tailwindlabs/heroicons#446] [Ideas]
Love the icons! Would love to see a thumbtack icon. Thanks!

**Comment by till1993** [comment]
Would be awesome to have a thumbtack icon!

**Comment by ndri** [comment]
I added `pin` and `pin-slash` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons. Browse them here: 
- https://sidekickicons.com/?icon=pin
- https://sidekickicons.com/?icon=pin-slash
